### PR TITLE
Pr/286

### PR DESCRIPTION
**Pull request by wayneglicc**

### CODE REVIEW

**Comment by ai** [comment]
1. Can you add a small description with your motivation for these changes?
2. Seems like your PR is broken (it is based on old Browserslist commits and can’t be merged). Can you delete your fork from GitHub, fork again, and then add all your changes to docs/code again?

**Comment by wayneglicc** [comment]
> _No description provided._

File: index.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ var FLOAT_RANGE = /^\d+(\.\d+)?(-\d+(\.\d+)?)*$/
 var YEAR = 365.259641 * 24 * 60 * 60 * 1000
```
Comment (wayneglicc) [inline index.js:12]: fuck off

**Comment by KVROHITH098** [comment]
BRO REALLY

**Pull request closed without merging**